playbooks/portals-setup-following.yml

```diff
@@ -25,19 +24,32 @@
       args:
         apply:
           become: True
-    
-    - name: Include firewall setup
-      include_tasks: tasks/portal-firewall-setup.yml
+
+    # Need to set up firewall then apply ip tables after docker is installed
```

Comment:
This fails because the ip tables rely on docker which isn't installed until `portal-setup-server.yml` is executed.